src/avatar/avatar.component.html

```diff
@@ -1,6 +1,10 @@
 <div class="igx-avatar igx-avatar--{{size}}" aria-label="avatar" role="img">
-    <img #image *ngIf="src || initials" src="{{src}}" class="igx-avatar--image igx-avatar--{{size}}" [class.igx-avatar--rounded]="isRounded"
-        [style.backgroundColor]="bgColor" [attr.aria-roledescription]="roleDescription" />
+    <img #image *ngIf="src" src="{{src}}" class="igx-avatar--image igx-avatar--{{size}}" [class.igx-avatar--rounded]="isRounded"
+         [style.backgroundColor]="bgColor" [attr.aria-roledescription]="roleDescription" />
+        <svg #initialsImage *ngIf="initials && !src" xmlns="http://www.w3.org/2000/svg" [class.igx-avatar--rounded]="isRounded" [style.backgroundColor]="bgColor"
+            [attr.aria-roledescription]="roleDescription">
+            <text text-anchor="middle" />
+        </svg>
```

Comment:
Indentation misalignment for the SVG element - should be 4 instead of 8. Also bind a additional classes to the SVG element of "igx-avatar--initials and igx-avatar--{{size}}" so that I can use it as a selectors when styling the component.